packages/block-editor/src/components/block-list/block.js

```diff
@@ -152,7 +152,9 @@ function BlockListBlock( {
 			isSelectionEnabled={ isSelectionEnabled }
 			toggleSelection={ toggleSelection }
 			__unstableLayoutClassNames={ layoutClassNames }
-			__unstableParentLayout={ parentLayout }
+			__unstableParentLayout={
+				Object.keys( parentLayout ).length ? parentLayout : undefined
```

Comment:
Is it possible for `parentLayout` to be `null` or `undefined`? If so `Object.keys()` will throw an exception here.

Reply:
I don't _think_ so, but perhaps it's best to explicitly make it default to an empty object to be on the safe side.